pkg/client/get_options.go

```diff
@@ -20,29 +20,60 @@ import (
 	"github.com/codenotary/immudb/pkg/api/schema"
 )
 
+// GetOption is used to set additional options when reading a value with a Get call
 type GetOption func(req *schema.KeyRequest) error
 
+// NoWait option set to true means that the server should not wait for the indexer
+// to be up-to-date with the most recent transaction.
+//
+// In practice this means that the user will get the result instantly without the
```

Comment:
Just confirming this for my understanding, this implies that if NoWait is set, the server will return the result instantly without waiting on the server for indexing, right?

Reply:
yes, exactly

Reply:
ack, thank you